geoportailv3/static/js/layermanager/layermanagerdirective.js

```diff
@@ -45,4 +45,23 @@ app.LayermanagerController.prototype.removeLayer = function(layer) {
 };
 
 
+/**
+ * @param {ol.layer.Layer} layer Layer.
+ * @export
+ */
+app.LayermanagerController.prototype.changeVisibility = function(layer) {
+  var currentOpacity = parseFloat(layer.get('opacity'));
+  var newOpacity;
+  if (currentOpacity === 0) {
+    newOpacity = layer.get('oldOpacity') ? layer.get('oldOpacity') : 1;
```

Comment:
Are you testing for the case where the old opacity is `0` or `undefined` here? I think we should make the code for explicit. For example:

```js
    var oldOpacity = layer.get('oldOpacity');
    newOpacity = goog.isDef(oldOpacity) ? oldOpacity : 1;
    // …
```